main/main.cpp

```diff
@@ -2176,6 +2178,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	// GLOBAL_DEF("xr/openxr/in_editor", false);
 #endif
 
+	// Joypad settings.
+	GLOBAL_DEF_BASIC("input_devices/joypad/enabled", enable_joypad);
```

Comment:
This will make the setting dependent on the command line, I don't think that's a good idea

Reply:
Indeed, project settings defaults shouldn't change based on CLI arguments or they won't be serialized to `project.godot` if the current setting matches the current default. This is why defaults should always be 100% static.

Reply:
That's an interesting question, I was thinking what will happen if one start the editor with --disable-joypad but open it in project setting manually, I guess I have to store the info about commanline someother places instead of reusing project setting.

Reply:
FYI: `--rendering-method <method>` is used if available. It otherwise uses the project setting `rendering/renderer/rendering_method` which has its own default value.

Reply:
Thanks for all your suggestions, I updated the code, it should feel more intuitive now.